packages/dashboard/src/components/account-button-toggle/accounts-button-toggle.scss

```diff
@@ -0,0 +1,31 @@
+.button-toggle {
+    &__container {
+        width: 20.8rem;
+        height: 4.8rem;
+        margin: 3rem auto;
+        padding: 0.8rem;
+        background-color: var(--general-section-1);
+        border-radius: 0.8rem;
+    }
+    &__items {
+        .dc-button-menu__wrapper {
+            background-color: unset !important;
+        }
+        .dc-button-menu--highlight {
+            background-color: $color-apple-blue !important;
+            color: $color-white !important;
```

Comment:
Do we keep the same colors in dark mode as well? If not, we should add it to `themes.scss` and get a theme color from there.

Reply:
There is no dark theme for DWallet.

Reply:
please add new one in themes.scss and make it the same color for light and dark theme @akmal-binary

Reply:
it would be easier to migrate later when there is dark mode

Reply:
@mustofa-binary this is outdated, code has been updated since then 😃